### PR TITLE
[MBL-16956][Student] Rubric LGO

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: test rubric with and without restrict quantitative data.

refs: MBL-16956
affects: Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/utils/Const.kt [hunk 1]
```diff
@@ -135,6 +135,7 @@ object Const {
     const val QUIZ_ANSWER_ID = "answer_id"
     const val ANONYMOUS_GRADING = "anonymousGrading"
     const val UNSUPPORTED_DESCRIPTION = "unsupportedDescription"
+    const val RESTRICT_QUANTITATIVE_DATA = "restrictQuantitativeData"
```
Comment (tamaskozmer) [inline libs/pandautils/src/main/java/com/instructure/pandautils/utils/Const.kt:138]: I am not sure this needs to be a public constant, it is only used in one class.

**Review by tamaskozmer — APPROVED** [review]
QA + 1

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**